src/hal/src/pm_stm32f4.c

```diff
@@ -504,3 +516,19 @@ LOG_ADD_CORE(LOG_UINT8, batteryLevel, &batteryLevel)
 LOG_ADD(LOG_FLOAT, temp, &temp)
 #endif
 LOG_GROUP_STOP(pm)
+
+/**
+ * Power management parameters.
+ */
+PARAM_GROUP_START(pm)
+/**
+ * @brief At what voltage power management will indicate low battery.
+ */
+PARAM_ADD_CORE(PARAM_FLOAT | PARAM_PERSISTENT, LowVoltage, &batteryLowVoltage)
```

Comment:
Most of the parameters in the firmware start with a lowercase letter (even if they are camelCased). This one is uppercase, I'm not sure whether it's intentional.

Reply:
Good one, I'll change it to camel case.